packaging/rhel/sogo.spec

```diff
@@ -49,6 +49,11 @@ BuildRequires:  gcc-objc gnustep-base gnustep-make sope%{sope_major_version}%{so
 %{?el8:Requires: liboath}
 %{?el8:BuildRequires: liboath-devel}
 
+%if 0%{?rhel} >= 7
```

Comment:
libsodium doesn't seem to be available on CentOS 8

Reply:
Thats odd. See https://centos.pkgs.org/8/epel-x86_64/libsodium-1.0.18-2.el8.x86_64.rpm.html